doc/command_guidelines.md

```diff
@@ -6,13 +6,27 @@ Guidelines marked (*) only apply to command modules, not extensions.
 
 If in doubt, ask!
 
```

Comment:
Should we recommend that commands from brand new modules should generally be marked as preview? Also that params should not end with '-resource-group'. 

These were some command guideline issues we discussed when discussing new possible linter rules

Reply:
That commands should not end with resource group is already captured here:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-name-or-id-parameters

Reply:
Statements regarding preview would be best located in the onboarding guide. The command authoring guide already has guidance on marking things preview (the how, not the when).

Reply:
I linked the onboarding guide to the authoring guide and added the guidance on starting things in Preview as well as what is required to transition from preview to GA, extension to module, etc.